java/.ci/Jenkinsfile

```diff
@@ -769,7 +769,28 @@ pipeline {
                     }
                 }
             }
-            
+            stage('Parallel Execution 10'){
+                                        parallel{
+                                            stage('PubSubLite To BigTable') {
+                                                steps{
+                                                    sh '''
+                                                        export GCP_PROJECT=yadavaja-sandbox
+                                                        export REGION=us-west1
+                                                        export GCS_STAGING_LOCATION="gs://python-dataproc-templates-temp"
+                                                        cd java
+                                                        bin/start.sh \
+                                                        -- --template PUBSUBLITETOBIGTABLE \
+                                                        --templateProperty pubsub.input.project.id=yadavaja-sandbox \
+                                                        --templateProperty pubsub.input.subscription=projects/617357862702/locations/us-west1/subscriptions/pubsublitetobigtable-test-subscription2 \
```

Comment:
Substitute, 617357862702 with $GCP_PROJECT_NUMBER

Reply:
Changed and committed